templates/webapi/main/build_progressbar.html.ep

```diff
@@ -1,4 +1,6 @@
-<div class="progress build-dashboard" title="<%= build_progress_bar_title($result) %>">
+% my $class = stash '';
```

Comment:
that does not look like you use it correctly. I thought the idea was that you set a distinct value (even "undef") in the controller when rendering the template